evalml/automl/automl_algorithm/evalml_algorithm.py

```diff
@@ -129,7 +129,7 @@ def __init__(
     def _naive_estimators(self):
         if is_regression(self.problem_type):
             naive_estimators = [
-                "Linear Regressor",
+                "Elastic Net Regressor",
```

Comment:
Hm so were we not running elastic net in automl before this point? That doesn't match with my memory but perhaps I've just forgotten the details of the plumbing here, lol

Reply:
This is on the new algorithm @jeremyliweishih wrote so not what's run by default in AutoMLSearch right now. In AutoMLSearch we run elastic net by default!